pyth-sdk/schema/price_feed.json

```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "PriceFeed",
```

Comment:
It would be nice if we didn't have to copy-paste these into the `pyth-sdk-js` repo, but I guess there's no easy way of achieving that without a monorepo 😉

Reply:
yeah... :( I added a description ro readme for the time being.